collectors/channels_collector.go

```diff
@@ -174,10 +184,55 @@ func NewChannelsCollector(lnd lndclient.LightningClient, errChan chan<- error,
 			[]string{"amount"}, nil,
 		),
 
-		lnd:         lnd,
-		primaryNode: cfg.PrimaryNode,
-		errChan:     errChan,
+		lnd:                 lnd,
+		primaryNode:         cfg.PrimaryNode,
+		closedChannelsCache: nil,
+		errChan:             errChan,
+		quit:                quitChan,
+	}
+
+	// Perform an initial refresh for the cache so there is a value to scrape.
```

Comment:
Since we now call this immediately within the goroutine (and not just after the first timer tick), I think we can remove this initial call.